src/renderer/browser/components/RequestsProcessor/Send/index.js

```diff
@@ -48,19 +50,21 @@ export default function makeSend(sendActions) {
       title: 'Transfer'
     }),
 
-    // Get the current network & account data
+    // Get the current network, fee settings data, & account data
     withNetworkData(),
+    withData(feeActions, mapFeeDataToProps),
     withData(authActions, mapAuthDataToProps),
 
     // Send assets & wait for success or failure
-    withInitialCall(sendActions, ({ net, amount, asset, receiver, address, wif, remark }) => ({
+    withInitialCall(sendActions, ({ net, amount, asset, receiver, address, wif, remark, fee }) => ({
       net,
       amount,
       asset,
       receiver,
       address,
       wif,
-      remark
+      remark,
+      fee
```

Comment:
You seem to fix the usage of fee/fees throughout the whole codebase. Could we use 1 constant for this? :P

Reply:
There's some cut-off point where we refer to it as "fee" but track it as "fees".  For example, in the UI we call it "Priority Fee".  That was the what I used as the decision for us to store it as `fee` instead of `fees`.  It's only when it gets into neon-js that it gets converted to `fees`.

I would prefer we use language that represents nOS usage vs. a what a third party library prefers, especially considering the library can change at any given time.